covid_model/analysis/run_mock_immunity.py

```diff
@@ -6,16 +6,21 @@
 from covid_model.db import db_engine
 from covid_model.model import CovidModel
 from covid_model.cli_specs import ModelSpecsArgumentParser
+from covid_model.analysis.charts import modeled
 
 
 def build_default_model(days):
-    model = CovidModel(engine=engine, **argparser.specs_args_as_dict())
-    model.update_specs(end_date=model.start_date + dt.timedelta(days=days))
-    model.build_param_lookups(apply_vaccines=False)
+    base_model = CovidModel(engine=engine, **argparser.specs_args_as_dict())
+    model = CovidModel(base_model=base_model, end_date=base_model.start_date + dt.timedelta(days=days))
+    model.build_param_lookups()
     model.set_param('shot1_per_available', 0)
     model.set_param('shot2_per_available', 0)
     model.set_param('shot3_per_available', 0)
     model.set_param('betta', 0)
+    model.set_param('initial_seed', 0)
```

Comment:
To deal with some issues with the y0 implementation, I changed y0 to always have everyone in susceptible, and then I add the "initial seed" with a parameter, like with the other variants.